notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>Line #4.    model_path = f"model/public/{model_name}/{precision}/{model_name}.xml"</p><p>to be more consistent, you can reuse downloaded_model_path here as part of model path</p><p> </p><p>model_path = f"{downloaded_model_path}/{precision}/{model_name}.xml"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>